packages/experimental/Badge/src/Badge.types.ts

```diff
@@ -10,40 +10,18 @@ export type BadgeAppearance = 'filled' | 'outline' | 'tint' | 'ghost' | 'filledI
 export type BadgeShape = 'rounded' | 'circular' | 'square';
 
 export interface BadgeTokens extends LayoutTokens, FontTokens, IBorderTokens, IShadowTokens, IColorTokens {
-  /**
```

Comment:
Is there a reason to remove all these comments? We will need to add them back when we add documentation for our props and tokens

Reply:
I think it can be in readme.md. I removed it, because these comments were obvious and just piles up the code

Reply:
I'm more of a fan of the code being the source of truth for documentation, since code is looked at moreso than documentation from developers. Ideally we'd have a system where the documentation is generated form the code but I don't think we found a good solution. I also like the inline documentation because intelligence on VSCode picks it up when you are typing.

Reply:
I agree with Saad, the types files are a public interface we ship and expose to clients, so we should prioritize documentation and information to clients over code readability in this file. Being able to provide documentation through intellisense is also good because then clients don't have to context switch to our documentation in order to read about what options are available for a prop, as an example.

Reply:
Ok, I'll return the comments

Reply:
Fixed